src/fields/field.impl.hh

```diff
@@ -642,7 +641,10 @@ void Field<spacedim,Value>::compute_field_data(OutputTime::DiscreteSpace space_t
     }
     break;
     case OutputTime::ELEM_DATA: {
-    	std::shared_ptr<OutputMesh> output_mesh = std::dynamic_pointer_cast<OutputMesh>( stream->get_output_mesh_ptr() );
+        std::shared_ptr<OutputMeshBase> output_mesh = stream->get_output_mesh_ptr(true);
```

Comment:
Does it mean that output on refined mesh is supported only for ELEM_DATA? What about NODE_DATA, CORNER_DATA, NATIVE_DATA.